src/pages/Governance/Proposal/card/VoteButtons.tsx

```diff
@@ -151,15 +162,29 @@ export default function VoteButtons({proposalId}: Props) {
   return (
     <>
       <Stack spacing={2}>
-        <TextField
-          fullWidth
-          label="Owner Account Address"
-          variant="filled"
-          size="small"
-          margin="normal"
-          value={accountAddr}
-          onChange={onAccountAddrChange}
-        />
+        {accountAddrIsValid ? (
```

Comment:
can we globalize those validation TextField ? looks like we use it here and also on the Stake page

Reply:
I didn't globalize them because they had different variant/size... But on a second thought, I might be good to make them the same variant/size to be consistent. I'll make the change.